doc/Building.md

```diff
@@ -20,7 +20,7 @@ functionalities such as rendering, serialization, file handling and network
 communication. Those dependencies are:
 * Boost 1.54 or later
 * MPI with MPI_THREAD_MULTIPLE support (openmpi 1.6.5 or later recommended)
-* Qt 5.4 or 5.5 (versions 5.6 and 5.7 currently have some
+* Qt 5.4, 5.5 or 5.7 (version 5.6 currently has some
```

Comment:
Yes, I tested locally and on the displaywall and didn't have any crash (whereas the 5.6 crashes in less than a minute). This is very reassuring, but I am still going to see if something can be done about the 5.6 builds.